mage/how-to-guides/run-mage-modules-on-subgraphs.md

```diff
@@ -0,0 +1,139 @@
+MATCH (n:Tweet {id: 1}) MERGE (n)<-[:RETWEETED]-(:Tweet {hashtag: "#WorldCup", text: "Farewall Dani Alves!"});
+MATCH (n:Tweet {id: 2}) MERGE (n)<-[:RETWEETED]-(:Tweet {hashtag: "#WorldCup", text: "This is the best WC ever!"});
+MATCH (n:Tweet {id: 2}) MERGE (n)<-[:RETWEETED]-(:Tweet {hashtag: "#WorldCup", text: "It's not so hot this time of the year in Qatar."});
+MATCH (n:Tweet {id: 2}) MERGE (n)<-[:RETWEETED]-(:Tweet {hashtag: "#WorldCup", text: "What are your predictions?"});
+MATCH (n:Tweet {id: 3}) MERGE (n)<-[:RETWEETED]-(:Tweet {hashtag: "#christmas", text: "Don't be a Grinch!"});
+MATCH (n:Tweet {id: 4}) MERGE (n)<-[:RETWEETED]-(:Tweet {hashtag: "#christmas", text: "I'll give you a present!"});
+MATCH (n:Tweet {id: 4}) MERGE (n)<-[:RETWEETED]-(:Tweet {hashtag: "#christmas", text: "Santa Claus will visit you tonight!"});
+MATCH (n:Tweet {id: 4}) MERGE (n)<-[:RETWEETED]-(:Tweet {hashtag: "#christmas", text: "This year save me from tears"});
+MATCH (n:Tweet {id: 4}) MERGE (n)<-[:RETWEETED]-(:Tweet {hashtag: "#christmas", text: "All I want for Christmas is youuuu"});
+```
+
+### The usual way of running PageRank on the whole network
+
+The usual way one would run a PageRank in MAGE is:
+
+```cypher
+CALL pagerank.get() YIELD node, rank
+SET node.rank = rank;
+```
+
+However, this kind of PageRank takes into account all of the nodes. It would be unwise to correlate tweets from the
+World Cup with tweets for Christmas, as they are not the same thing, and a plausible case would be if they
+could be separated.
+
+### Running PageRank on a subgraph
+
+With a small modification, by putting the graph projection in as the first argument of the query module, it is 
+executed on the subgraph, like in the query below:
+
+```cypher
+MATCH p=(n:Tweet {hashtag: "#christmas"})-[r]->(m)
+WITH project(p) AS christmas_graph
+CALL pagerank.get(christmas_graph) YIELD node, rank
+SET node.rank = rank
+RETURN node.hashtag, node.text, rank
+ORDER BY rank DESC;
+```
+
+<img src={require('../data/how-to-guides/subgraphs-guide/christmas-subgraph.png').default}/>
+
+With this query, we have successfully updated the rank of the Christmas tweets only! Let's do the same
+on the World Cup tweets:
+
+```cypher
+MATCH p=(n:Tweet {hashtag: "#WorldCup"})-[r]->(m)
+WITH project(p) AS christmas_graph
+CALL pagerank.get(christmas_graph) YIELD node, rank
+SET node.rank = rank
+RETURN node.hashtag, node.text, rank
+ORDER BY rank DESC;
+```
+
+<img src={require('../data/how-to-guides/subgraphs-guide/world-cup-subgraph.png').default}/>
+
+The query is basically identical, with only the projection changing from one hashtag to another.
+
+### Final remarks
+Thanks for reading our how to guide on how to run subgraph query modules in MAGE!
+If you want to try out some of our algorithms in subgraph mode, you can pick one from our [list of available query modules](/mage/query-modules/available-queries)!
```

Comment:
```suggestion
This how-to guide showed how to run algorithms on subgraphs. If you want to try out other MAGE algorithms and run them on subgraphs, check the [list of available query modules](/mage/query-modules/available-queries)!
```

Reply:
Should we add something like try it by installing platform or checking out cloud or something like that?